File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio3/Python/cliente.py
class Cliente:
    def __init__(self, id: int, nombre: str, celular: str):
        self.id = id
        self.nombre = nombre
        self.celular = celular

    def __str__(self):
        return f"Cliente(id={self.id}, nombre='{self.nombre}', celular='{self.celular}')"

File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio3/Java/Cliente.java
package PERSISTENCIA.Ejercicio3.Java;

import java.io.Serializable;

public class Cliente implements Serializable {
    private int id;
    private String nombre;
    private String celular;

    public Cliente(int id, String nombre, String celular) {
        this.id = id;
        this.nombre = nombre;
        this.celular = celular;
    }

    public int getId() { return id; }
    public String getNombre() { return nombre; }
    public String getCelular() { return celular; }

    @Override
    public String toString() {
        return "Cliente{" + "id=" + id + ", nombre='" + nombre + '\'' +
               ", celular='" + celular + '\'' + '}';
    }
}
